tests/unit/test_expression_tree/test_operations/test_jac.py

```diff
@@ -10,7 +10,7 @@
 from tests import get_mesh_for_testing
 
 
-def test_multi_var_function(arg1, arg2):
+def multi_var_function_test(arg1, arg2):
```

Comment:
Seems like these functions repeat a lot, can we move them to a utilities file instead of just renaming them all. Good to clean up a bit when stuff like this is found

Reply:
in `util.py` file right?

Reply:
Yeah in a testing utils file, not a main pybamm utils file. Cutting down on duplicated code is just a good thing, especially when you already had to change the names of them all

Reply:
For reference, `tests/shared.py` is what we've been using for this purpose

Reply:
Done!